resources/epilepsy/ResourceFile_Epilepsy/Structure, parameters, refs.csv

```diff
@@ -0,0 +1,3 @@
+version https://git-lfs.github.com/spec/v1
+oid sha256:e5f0e3c22d3b82061dc884df42f3a304dd0cd1b205166d1e5460c8569eba0d53
```

Comment:
similarly if this is just documentation r.e. assumptions maybe it doesnt need to live on master?

Reply:
should I delete it then? I will defer to you or @andrew-phillips-1 for a final decision

Reply:
Hi @mnjowe It's fine to delete from here so long as we have a central copy kept somewhere.  I can't recall where the write-ups are held.

Reply:
great, thanks @andrew-phillips-1. For all the files I have converted, original copies are stored in our TLO sharepoint [here](https://imperiallondon.sharepoint.com/:f:/r/sites/tlomodel-WP/Shared%20Documents/08%20-%20Archive/Excel%20files%20converted%20to%20CSV?csf=1&web=1&e=I7EPp3)